src/objects/api/v2/openapi.yaml

```diff
@@ -457,6 +457,53 @@ paths:
       responses:
         '204':
           description: No response body
+  /objects/{uuid}/{index}:
+    get:
+      operationId: object_history_detail
+      description: Retrieve the specified OBJECT given an UUID and INDEX.
+      parameters:
+      - in: header
+        name: Accept-Crs
+        schema:
+          type: string
+          enum:
+          - EPSG:4326
+        description: 'The desired ''Coordinate Reference System'' (CRS) of the response
+          data. According to the GeoJSON spec, WGS84 is the default (EPSG: 4326 is
+          the same as WGS84).'
+      - in: path
+        name: index
+        schema:
+          type: string
+          pattern: ^\d+$
```

Comment:
Is this correct? I think this should be a positive integer instead of a string

Reply:
See https://github.com/maykinmedia/objects-api/pull/429/commits/a9a808aae65bfa80290bed133fddb83df224d002